examples/controls_example/public/react_controls/data_controls/range_slider/get_range_slider_control_factory.tsx

```diff
@@ -61,7 +61,7 @@ export const getRangesliderControlFactory = (
       const loadingMinMax$ = new BehaviorSubject<boolean>(false);
       const loadingHasNoResults$ = new BehaviorSubject<boolean>(false);
       const dataLoading$ = new BehaviorSubject<boolean | undefined>(undefined);
-      const step$ = new BehaviorSubject<number | undefined>(initialState.step ?? 1);
+      const step$ = new BehaviorSubject<number | undefined>(initialState.step);
```

Comment:
Can we also change the comparator + set the default here to keep things consistent? And bring the deleted test back 🫡

Reply:
resolved in e938b08d8af67f4b452f0ebb0414ca00d790f787